features/steps/generic.py

```diff
@@ -34,12 +34,6 @@ def check_title(context, title):
     page_title = WebDriverWait(driver, timeout=timeout).until(lambda d: d.find_element(By.TAG_NAME, "h1")).text
     assert page_title == title
 
-@then('The title of the help centre page is "{title}"')
```

Comment:
This test is identical to the test immediately above it, except that it searches for the "h3" tag rather than the "h1" tag. This PR involves removing the h3 tag, and changes the h1 tag to make it unique for each help centre page. This means that the h1 test should be used instead, and the h3 test is no longer needed as it is not used.

Reply:
Sounds good =) All fine by me!